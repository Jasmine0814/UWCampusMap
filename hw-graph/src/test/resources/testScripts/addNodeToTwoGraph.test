# test that create two graphs and add 3 nodes in graph1, 2 nodes in graph2, 1 node in graph3.

# Create a graph
created graph graph1
created graph graph2
created graph graph3

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph2 n4
AddNode graph2 n5
AddNode graph3 n6

# Print the nodes in the graphs
ListNodes graph1
ListNodes graph2
ListNodes graph3
