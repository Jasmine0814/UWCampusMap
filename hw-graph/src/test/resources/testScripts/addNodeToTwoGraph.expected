# test that create two graphs and add 3 nodes in graph1, 2 nodes in graph2, 1 node in graph3.

# Create a graph
created graph graph1
created graph graph2
created graph graph3

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph2
added node n5 to graph2
added node n6 to graph3

# Print the nodes in the graphs
graph1 contains: n1 n2 n3
graph2 contains: n4 n5
graph3 contains: n6



